cmk/gui/plugins/views/commands.py

```diff
@@ -1163,7 +1163,7 @@ def _recurring_number(self):
 
     def _flexible_option(self):
```

Comment:
I think the function is not actually checked unless you add the return type here.

Reply:
Added.